validator/src/commands/run/args.rs

```diff
@@ -774,25 +774,19 @@ pub fn add_args<'a>(app: App<'a, 'a>, default_args: &'a DefaultArgs) -> App<'a,
             .validator(is_parsable::<u64>)
             .help("Milliseconds to wait in the TPU receiver for packet coalescing."),
     )
-    .arg(
-        Arg::with_name("tpu_use_quic")
```

Comment:
why was this removed without deprecation? sure it's the "default" but it may well still be in validator configs, which will now fail on unexpected arg on startup